package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rcs-webpack-plugin",
-  "version": "1.4.2",
+  "version": "1.5.0",
```

Comment:
Please do not upgrade the version

Reply:
Reverted the change [here](https://github.com/JPeer264/rcs-webpack-plugin/pull/23/commits/0ba148e23e96f9bcb0b65f03e80203b6df1de0a0)